zombienet.sh

```diff
@@ -9,4 +9,19 @@ Available at https://github.com/paritytech/zombienet"
   exit 1
 fi
 
+if ! [ -x "$(command -v polkadot)" ]; then
+  echo "\
+'polkadot' is not found in PATH. Install polkadot.
+
+To obtain, refer to https://github.com/paritytech/polkadot-sdk/tree/master/polkadot#polkadot"
```

Comment:
we also need the prepare-worker and execute-worker, AFAIK

Reply:
Yeah, that's true. I thought it should be handled by the instructions for the preferred installation method. For pre-compiled binaries it specifically mentions:

> make sure to also download all the worker binaries and put them in the same directory as polkadot

Or do you suggest detecting those binaries present in PATH? I checked it, and it seems that being in PATH doesn't affect anything. Polkadot looks for the binaries either in the same directory it was invoked from or in /usr/lib/polkadot